docs/book/src/developer/tilt.md

```diff
@@ -131,6 +131,27 @@ base64 -i ~/path/to/gcp/credentials.json
 **trigger_mode** (String, default=`auto`): Optional setting to configure if tilt should automatically rebuild on changes.
 Set to `manual` to disable auto-rebuilding and require users to trigger rebuilds of individual changed components through the UI.
 
+**extra_args** (Object, default={}): A mapping of provider to additional arguments to pass to the main binary configured
+for this provider. Each item in the array will be passed in to the manager. 
```

Comment:
Is `for this provider` still accurate once this content is moved to this section?

Reply:
Rolling with @JoelSpeed's question, maybe `for the given provider` would be more accurate instead?

Reply:
fixed as per @detiber suggestion